packages/harpguru-core/src/components/toggle-buffer-flusher/toggle-buffer-flusher.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import { useFlushBufferedActivityToggles } from './hooks'
+
+export const ToggleBufferFlusher = (): React.ReactElement => {
```

Comment:
It isn't necessary to have this hook called from a component. I wonder whether it should be moved back to just being called from the harp-guru-page object.

Reply:
Recorded as techdebt in https://github.com/js-jslog/harpguru/issues/52#issuecomment-737527652

Reply:
I'm taking no action on this. I like the reduced clutter in the pre-return component body.